app/code/RLTSquare/Ccq/etc/di.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\Framework\Console\CommandListInterface">
+        <arguments>
+            <argument name="commands" xsi:type="array">
+                <item name="rltsquareccq_helloworldcommand" xsi:type="object">
+                    RLTSquare\Ccq\Console\Command\HelloWorld
```

Comment:
Code refactoring - sometimes breaking XML like this, can break code, sometime silently, so it is hard to debug code like this. This should be kept single line.